tests/Browser/Pages/Backend/Categories/AdminEditCategoryTest.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+namespace Tests\Browser\Pages\Backend\Cattegories;
+
+use App\Model\User;
+use App\Model\Category;
+use Tests\DuskTestCase;
+use Laravel\Dusk\Browser;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Faker\Factory as Faker;
+use DB;
+
+class AdminEditCategoryTest extends DuskTestCase
+{
+    use DatabaseMigrations;
+
+    const DEFAULT_CATEGORY = 'Default Category';
+    const NUMBER_RECORD_CREATE = 5;
+
+    protected $buttonSelected = '.item-2 .btn-edit-category';
+    protected $inputSelected = '.item-2 input';
+    protected $textSelected = '.item-2 p';
+
+    /**
+     * Override function setUp()
+     *
+     * @return void
+     */
+    public function setUp()
+    {
+        parent::setUp();
+        factory(Category::class)->create([
+            'title' => self::DEFAULT_CATEGORY
+        ]);
+        $this->makeData(self::NUMBER_RECORD_CREATE);
+    }
+
+    /**
+     * Test view Admin Edit button
+     *
+     * @return void
+     */
+    public function testButtonEditCategory()
+    {
+        $this->browse(function (Browser $browser) {
+            $browser->loginAs($this->user)
+                ->visit('/admin/categories')
+                ->resize(1200,1600)
+                ->assertSee('List Categories');
+            $elements = $browser->elements('.btn-edit-category');
+            $this->assertCount(self::NUMBER_RECORD_CREATE, $elements);
+        });
+    }
+
+    /**
+     * Test click Edit button
+     *
+     * @return void
+     */
+    public function testClickButtonEditCategory()
+    {
+        $this->browse(function (Browser $browser) {
+            $browser->loginAs($this->user)
+                ->visit('/admin/categories')
+                ->resize(1200,1600);
+            $text = $browser->text($this->textSelected);
+            $browser->press($this->buttonSelected)
+                ->assertInputValue($this->inputSelected, $text);
+        });
+    }
+
+    /**
+     * Test show popup confirm edit
+     *
+     * @return void
+     */
+    public function testShowConfirmEditCategory()
+    {
+        $this->browse(function (Browser $browser) {
+            $browser->loginAs($this->user)
+                ->visit('/admin/categories')
+                ->resize(1200,1600)
+                ->press($this->buttonSelected)
+                ->keys($this->inputSelected, '{enter}')
+                ->assertSee('Confirm edit!')
+                ->assertSee('Do you want edit from');
```

Comment:
`Do you want edit from`? Chữ này sai rồi.
Change thành `Do you want edit this category?`